### PR TITLE
Dipole Annotation Mode

### PR DESCRIPTION
**Pull request by aysecanuenal**
Dipole annotation mode added here so we can add points and vectors in the tomogram

### CODE REVIEW

File: src/napari_tomoslice/_constants.py [hunk 1]
```diff
@@ -42,3 +42,11 @@
 - press 'r' at desired edge of sphere to change the radius
 - {_ALT} + click again to move the sphere 
 """
+
+DIPOLE_ANNOTATOR_HELP_TEXT = f"""
+- press 'n' to start a new dipole
+- {_ALT} + click to set the position of the new dipole
+- press 'v' to set the direction of the dipole
```
Comment (alisterburt) [inline src/napari_tomoslice/_constants.py:49]: I've been second guessing this - not sure why but it didn't feel intuitive when I was playing with it. How do we feel about 'd' for direction instead? 🙂 if you agree I'll update n3d and push a release

**Review by alisterburt — APPROVED** [review]
@aysecanuenal this is wonderful. The code is super clean, this is only possible because of the great work you did over at n3d - you should feel proud 🙂

I have a minor nit about the keybinding but once we've decided how to handle that let's merge and deal with it in a follow up!

**Comment by alisterburt** [comment]
Test failures are unrelated

File: src/napari_tomoslice/_constants.py (hunk 1 above)
Comment (aysecanuenal) [reply src/napari_tomoslice/_constants.py:49]: Yes, I like that!

**Pull request merged**